### ISSUE TITLE
Standard output and Standard error are not being written unless docker is invoked with -i.

### ISSUE THREAD

**Issue by git-rz** [rancher-sandbox/rancher-desktop#3239]
### Actual Behavior

In WSL only, Standard output and Standard error are not being written unless docker is invoked with -i.
I have tried from within several distros: Ubuntu 22, 20, 18, Kali.

### Steps to Reproduce

Must be run in a WSL distro:

```
docker run -i --rm alpine  echo "why do I need to add -i to get my output? This makes rancher-desktop on WSL unusable"

docker run  --rm alpine  echo "why do I need to add -i to get my output? This makes rancher-desktop on WSL unusable"
```

### Result

![image](https://user-images.githubusercontent.com/53906153/197119202-6478fa95-a103-4f6d-9633-3b20c76900e4.png)

### Expected Behavior

Rancher-Desktop on Mac shows the output.
Rancher-Desktop on Windows from inside a CMD prompt shows proper output
Rancher-Desktop on Windows w/ nerdctl & containerd backend shows the output 
Docker-Desktop on Windows shows  the output.
Docker on Linux shows the output.

mac
![image](https://user-images.githubusercontent.com/53906153/197216161-4f0283b8-cb1e-4491-b5f3-4e4eb36df1b4.png)
git bash(zsh)
![image](https://user-images.githubusercontent.com/53906153/197215476-0d03b53e-a20d-4d1a-a4b0-8fbacaecfd47.png)
cmd
![image](https://user-images.githubusercontent.com/53906153/197215619-8bdc7d25-2f56-4057-a392-fad373368bf0.png)

### Additional Information

Ubuntu 22 in WSL
![image](https://user-images.githubusercontent.com/53906153/197217506-7ee31eb3-4c67-4a00-bc5f-1c31b164edfb.png)

### Rancher Desktop Version

1.6.0

### Rancher Desktop K8s Version

1.24.6

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 10 Enterprise 21H2

### What CPU architecture are you using?

x64

**Comment by jandubois** [comment]
See also https://github.com/kubernetes-sigs/kind/issues/3065

**Comment by sergioprates** [comment]
Any update?

**Comment by ivanmaia** [comment]
Any update?

**Comment by jandubois** [comment]
I'm sorry, this issue somehow got lost in the migration to the non-legacy Github projects. 😞 I thought we were tracking it on the new board already, but it seems like I mixed it up with the (related) #2094 issue.

Unfortunately, with summer vacations and other unexpected absences there is no way to get this dealt with in time for the 1.10 release, but I've added it to the 1.11 milestone now.

**Comment by ivanmaia** [comment]
Thank you!

**Comment by jandubois** [comment]
This looks like a duplicate of #1558.

**Comment by CalaxDev** [comment]
@jandubois I see you initially planned this for 1.11 but now removed it from the 1.13 milestone - What is the plan here? Adding args via docker run might work in a normal setting but not with predefined docker-compose files

**Comment by igieon** [comment]
I hope this will be soon solved. I get bitten it in perl plugin for idea. I try to solve it with fixing there but that seems wrong ...
https://github.com/Camelcade/Perl5-IDEA/pull/2836

**Comment by jandubois** [comment]
> I hoped that it would have been done in 1.13 already

Yeah, me too, but unfortunately we didn't have anybody available to work on it. It won't be fixed in 1.14 either, sorry!

**Comment by MrWako** [comment]
After being confused by the Rancher Desktop behaviour for a while finally realized I've hit this issue. Any chance it can get prioritized? Its really quite a blocker for serious usage with WSL2.

**Comment by MrWako** [comment]
Not suggesting this is a solution but in case it sheds any light, running via `nerdctl` on WSL2 works as expects and echoes the message.
```
nerdctl run --rm --address /var/run/docker/containerd/containerd.sock alpine echo "why do I need to add -i to get my output? This makes rancher-desktop on WSL unusable"
```
It does trip this [issue](https://github.com/rancher-sandbox/rancher-desktop/issues/3267) though.

**Comment by MrWako** [comment]
Nice analysis of the issue @matejkramny in your PR: https://github.com/rancher-sandbox/rancher-desktop/pull/7878 and for identifying that the issue is with the Go reverse proxy: https://github.com/golang/go/issues/35892. I've re-worked a previously suggested fix that hopefully we can get returned to the Go mainline and should then just require a compiler update to fix the issue.

**Comment by jandubois** [comment]
I believe this issue has been fixed by #7896 and can be closed.

Please comment/reopen if I am mistaken!
